src/literal.cc

```diff
@@ -260,7 +271,9 @@ Result FloatParser<T>::ParseHex(const char* s,
   int significand_exponent = 0;  // Exponent adjustment due to dot placement.
   for (; s < end; ++s) {
     uint32_t digit;
-    if (*s == '.') {
+    if (*s == '_') {
```

Comment:
Yeah, the code above only removes underscores for the `strtof`/`strtod` path. I guess I could remove them from other passes too, but that's the only case where it was necessary.